src/test/rustdoc-json/generic-associated-types/gats.rs

```diff
@@ -0,0 +1,19 @@
+// ignore-tidy-linelength
+
+#![no_core]
+#![feature(generic_associated_types, lang_items, no_core)]
+
+#[lang = "sized"]
+pub trait Sized {}
+
+pub trait LendingIterator {
+    // @count gats.json "$.index[*][?(@.name=='Item')].inner.generics.params" 1
+    // @is gats.json "$.index[*][?(@.name=='Item')].inner.generics.params[*].name" \"\'a\"
+    // @count gats.json "$.index[*][?(@.name=='Item')].inner.generics.where_predicates" 1
+    // @count gats.json "$.index[*][?(@.name=='Item')].inner.generics.where_predicates" 1
```

Comment:
These are the same. Do you want to check two things, or did you just duplicate a line?

Reply:
duplicated, I think...?